### PR TITLE
Start to add C# to VR starter tutorial

### PR DESCRIPTION
**Pull request by skyace65**
Adds C# for the first code block in the VR starter tutorial. I would have done more but I kept running into problems I don't know how to solve.

### CODE REVIEW

File: tutorials/vr/vr_starter_tutorial.rst [hunk 1]
```diff
@@ -116,6 +117,29 @@ First, let's get the VR up and going! While ``Game.tscn`` is open, select the ``
...
+
+    using Godot;
+    using System;
+
+    public class Game : Spatial
+    {
+        public override void _Ready()
+        {
+            var VR = ARVRServer.FindInterface("OpenVR");
+            if (VR != null && VR.Initialize())
```
Comment (neikeq) [inline tutorials/vr/vr_starter_tutorial.rst]: The naming convention for locals is camelCase, so this should be `vr`.

**Comment by skyace65** [comment]
Updated it to be camelCase

**Comment by BastiaanOlij** [comment]
Cool! So it all works fine in C#? Man that is exciting! thanks for updating the docs!

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**